lib/ansible_tower_client/base_models/inventory_source.rb

```diff
@@ -4,6 +4,10 @@ def self.endpoint
       "inventory_sources".freeze
     end
 
+    def self.create(_api, _attributes)
+      "POST not allowed"
```

Comment:
@bdunne - POST isn't allowed on that endpoint:

<img width="399" alt="screen shot 2017-01-09 at 9 12 36 am" src="https://cloud.githubusercontent.com/assets/697347/21769292/f88c6b82-d64b-11e6-8a3e-27b5c2d18dfd.png">

Would we rather allow that class to call `create` and depend on the API to raise the error, or save the lookup and raise the same error ourselves from the gem?